docs/sdk/src/guides/enable_elastic_scaling.rs

```diff
@@ -0,0 +1,238 @@
+//! # Enable elastic scaling for a parachain
+//!
+//! <div class="warning">This guide assumes full familiarity with Asynchronous Backing and its
+//! terminology, as defined in <a href="https://wiki.polkadot.network/docs/maintain-guides-async-backing">the Polkadot Wiki</a>.
+//! Furthermore, the parachain should have already been upgraded according to the guide.</div>
+//!
+//! ## Quick introduction to elastic scaling
+//!
+//! [Elastic scaling](https://polkadot.network/blog/elastic-scaling-streamling-growth-on-polkadot)
+//! is a feature that enables parachains to seamlessly scale up/down the number of used cores.
+//! This can be used to increase the compute or storage throughput of a parachain or
+//! to lower the latency between a transaction being submitted and it getting built in a parachain
+//! block.
+//!
+//! ## Current constraints
+//!
+//! Elastic scaling is still considered experimental software, so stability is not guaranteed.
+//! If you encounter any problems,
+//! [please open an issue](https://github.com/paritytech/polkadot-sdk/issues).
+//! Below are described the constraints of the implementation:
+//!
+//! 1. **Bounded compute throughput**. Each parachain block gets at most 2 seconds of execution on
+//!    the relay chain. Therefore, assuming the full 2 seconds are used, a parachain can only
+//!    utilise at most 3 cores in a relay chain slot of 6 seconds. If the full execution time is not
+//!    being used, or if collators are able to author blocks faster than the validators are able to
+//!    run them, higher core counts can be achieved.
+//! 2. **Single collator requirement for consistently scaling beyond a core at full authorship
+//!    duration of 2 seconds per block.** Using the current implementation with multiple collators
+//!    adds additional latency to the block production pipeline. Assuming block execution takes
+//!    about the same as authorship, the additional overhead is equal the duration of the authorship
+//!    plus the block announcement. Each collator must first import the previous block before
+//!    authoring a new one, so it is clear that the highest throughput can be achieved using a
+//!    single collator. Experiments show that the peak performance using more than one collator
+//!    (measured up to 10 collators) is utilising 2 cores with authorship time of 1.3 seconds per
+//!    block, which leaves 400ms for networking overhead. This would allow for 2.6 seconds of
+//!    execution, compared to the 2 seconds async backing enabled.
+//!    The development required for enabling maximum compute throughput for multiple collators is tracked by
+//!    [this issue](https://github.com/paritytech/polkadot-sdk/issues/5190).
```

Comment:
I think we can do much better in terms of structure here vs a large blob of text which is not that easy to read and focus important information.

Reply:
I rewrote this section. let me know how it looks